core/templates/dev/head/pages/exploration_editor/editor_tab/ResponsesService.js

```diff
@@ -107,7 +107,12 @@ oppia.factory('ResponsesService', [
       if (updates.dest) {
         answerGroup.outcome.dest = updates.dest;
       }
-      answerGroup.correct = false;
+      if (updates.labelledAsCorrect == true) {
```

Comment:
Drop "== true" -- it's not needed.

Better still, remove all this and just do:

```
if updates.hasOwnProperty('labelledAsCorrect') {
  answerGroup.labelledAsCorrect = updates.labelledAsCorrect;
}
```

Reply:
Done!